backend/lib/services/members/MemberManager.js

```diff
@@ -97,6 +97,19 @@ class MemberManager {
     return this.subjectList.members
   }
 
+  async rotateServiceAccountSecret (id) {
+    const item = this.subjectList.get(id)
+    if (!item) {
+      return
+    }
+
+    if (!item.kind === 'ServiceAccount') {
```

Comment:
Very strange kind of boolean algebra
```suggestion
    if (item.kind !== 'ServiceAccount') {
```

Reply:
Don't we have a unit test for this case?

Reply:
we have a test and if the expression above is wrong the test fails